src/internal/base-component/index.ts

```diff
@@ -1,49 +1,34 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { ALWAYS_VISUAL_REFRESH, PACKAGE_VERSION, THEME } from '../environment';
+import { metrics } from '../metrics';
+import { initAwsUiVersions } from './init-awsui-versions';
+import { useVisualRefresh } from './use-visual-refresh.js';
```

Comment:
I have lost tracking what is happening with all file moves. Let's get back to the task

1. `useBaseComponent` changes its location from this package to `@cloudscape-design/component-toolkit/internal`
2. Because it has dependency on `useVisualRefresh` hook, that hook moves to `@cloudscape-design/component-toolkit/internal` too.

After the move, we replace all imports in this in this package with their equivalents from `@cloudscape-design/component-toolkit/internal`.

Why is it needed to move that many files around? I am afraid that if we merge it the current way, the very next PR would require to undo some of the changes

Reply:
Thanks for the feedback - I will revert some changes to reduce the amount of diff and annotate the code that can be moved to the toolkit.